README.md

```diff
@@ -63,7 +63,9 @@ Users needing to import XML OSCAL artifacts are recommended to look at NIST's XM
 
 ## Python codebase, easy installation via pip
 
-Trestle runs on most all python platforms (e.g. Linux, Mac, Windows) and is available on PyPi so it is easily installed via pip.  It is under active development and new releases are made available regularly.
+Trestle runs on almost all Python platforms (e.g. Linux, Mac, Windows), is available on PyPi and can be easily installed via pip. It is under active development and new releases are made available regularly.\
```

Comment:
Are the '\' s at the end of lines 66 and 67 intentional?

Reply:
Yes, line breaks.

Reply:
Use a newline instead?

Reply:
The are in separate lines, but are rendered in a single line without a line break (`\`, `<br>`, or `&nbsp` )